.github/ISSUE_TEMPLATE/type_improvement.yml

```
@@ -1,5 +1,5 @@
 name: Improvement Request
-description: File a bug report
+description: File a improvement report
```

Shouldn't this be request? here and for task and new feature as well?
```suggestion
description: File a improvement request
```